Android/app/src/main/java/io/github/project_travel_mate/ProfileActivity.java

```diff
@@ -733,6 +733,7 @@ private void fillProfileOffline() {
         editDisplayName.setVisibility(View.INVISIBLE);
         editDisplayStatus.setVisibility(View.INVISIBLE);
         changeImage.setVisibility(View.INVISIBLE);
+        citiesTravelledHeading.setVisibility(View.INVISIBLE);
```

Comment:
you have hid it for every case. not just offline mode...

Reply:
I tested it in online mode and it visible because i just make it invisible in fillProfileOffline method and this method only work if no network connection